### PR TITLE
Pan European Header

### LINKED ISSUE #581: PanEuropean: App header
PanEuropean assessment header show:
left side: Home, language, separator

right side: Logos FAO (use black multilingual), UNECE, EUGL

### PR DESCRIPTION
**Pull request by sorja**
Resolves #581

### CODE REVIEW

**Comment by sorja** [comment]
Use dynamic component switcher

File: webapp/components/Header/FraHeader/FraHeader.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import LinkHome from '@webapp/components/Header/components/linkHome'
+import LanguageSelection from '@webapp/components/Header/components/languageSelection'
+import Logo from '@webapp/components/Header/Logo'
+import UserLinks from '@webapp/components/Header/UserLinks'
+
+const FraHeader = () => (
+  <div className="app-header no-print">
+    <Logo />
```
Comment (minotogna) [inline webapp/components/Header/FraHeader/FraHeader.js:9]: you can inline LogoFRA

File: webapp/components/Header/PanEuropeanHeader/PanEuropeanHeader.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  const i18n = useI18n()
+  return (
+    <div className="app-header pan-european no-print">
+      <LinkHome />
+      <LanguageSelection />
+      <div className="app-header__separator" />
+      <div className="app-header__global-fra">{i18n.t('panEuropean.panEuropeanAssessment')}</div>
+
+      <div className="app-header__menu">
+        <Logo />
```
Comment (minotogna) [inline webapp/components/Header/PanEuropeanHeader/PanEuropeanHeader.js:17]: you can inline LogoPanEuropean

**Review by minotogna — CHANGES_REQUESTED** [review]

File: webapp/components/Header/header.less [hunk 3]
```diff
@@ -9,6 +9,9 @@
   background: white;
   color: @text-mute;
   padding: 0 20px;
+  &.pan-european {
+    grid-template-columns: 27px calc(@ui-app-navigation-width - 46px) 1px auto auto;
+  }
```
Comment (minotogna) [inline webapp/components/Header/header.less:14]: why is this different for pan European ?

Comment (sorja) [reply webapp/components/Header/header.less:14]: Added more elements, needed one more column

Comment (sorja) [reply webapp/components/Header/header.less:14]: First one is the home icon, second one is the language switcher, the horizontal line, etc

Comment (sorja) [reply webapp/components/Header/header.less:14]: Otherwise, the "Pan european" text would collide with the logos of the right hand side

Comment (minotogna) [reply webapp/components/Header/header.less:14]: I thought Home button and language switcher could fit into the first column?

Comment (sorja) [reply webapp/components/Header/header.less:14]: I can wrap them in a div

**Review by minotogna — APPROVED** [review]

**Pull request merged**